.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,2 @@
+blank_issues_enabled: false
```

Comment:
I think we should allow blank issues... the GH ui pushes the templates hard, but sometimes you just want to start from scratch.